app/models/colyseus-models/pokemon.ts

```diff
@@ -13798,6 +13840,27 @@ export class WormadamTrash extends Pokemon {
       !regionSynergies.includes(Synergy.GRASS)
     )
   }
+
+  onEvolve({
```

Comment:
instead of duplicating code, point to a common reference. See what we did with `rksSystemOnChangePosition`